server/game/cards/23-AHaH/TheSeedIsStrong.js

```diff
@@ -35,7 +35,7 @@ class TheSeedIsStrong extends DrawCard {
 
     onCardSelected(context, card) {
         this.game.addMessage('{0} discards {1} from their {2}', context.event.winner, card, card.location);
-        GameActions.discardCard({ card });
```

Comment:
We should probably prefer to use game actions, rather than revert to the methods on the Player class. You should just be able to pass the game action to `this.game.resolveGameAction(...)` here.